release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1051,61 +1355,371 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+  
+* Catalog update pods with polling intervals that were multiples of 15 greater than the default 15 minute resynchronization period would be continuously reconciled by the Catalog Operator. This would continue until the next poll time was reached, causing increased CPU load. The reconciliation requeuing logic is now improved so that the continuous reconciliation and the associated CPU load increases do not occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1920526[*BZ#1920526*])
+
+* Previously, if no matching Operators were found during an attempt to create an Operator subscription, the constraints listed in the resolution failure event contained internal terminology. The subscription constraint strings did not describe the resolution failure reason from a user perspective. The constraint strings are now more meaningful. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921954[*BZ#1921954*])
+
+*openshift-apiserver*
+
+* Previously, requests targeting the `deploymentconfigs/<name>/instantiate` subresource failed with `no kind "DeploymentConfig" is registered for version apps.openshift.io/`. The correct version for the `DeploymentConfig` is now set and these requests no longer fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1867380[*BZ#1867380*])
+
+*Operator SDK*
+
+* Previously, all `operator-sdk` subcommands attempted to read the `PROJECT` file, even if `PROJECT` was a directory. As a result, subcommands that did not require the `PROJECT` file failed. Now, subcommands that do not require the `PROJECT` file do not attempt to read it and succeed even if an invalid `PROJECT` file is present. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1873007[*BZ#1873007*])
+
+* Previously, running the `operator-sdk cleanup` command did not clean up Operators that were deployed with the `operator-sdk run bundle` command. Instead, an error message was displayed and the Operator was not cleaned up. Now, the`operator-sdk cleanup` command has been updated, and Operators deployed with `run bundle` can be cleaned up by using the `cleanup` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883422[*BZ#1883422*])
 
 *Performance Addon Operator*
 
-* Previously, incorrect wait in the must-gather logic resulted in too early termination of log gathering. This issue resulted in depending on timing the log gathering operation being interrupted prematurely, leading to partial log collection.
-This is now fixed by adding the correct wait in the must-gather logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906355[*BZ#1906355*])
+* Previously, incorrect wait in the must-gather logic resulted in too early termination of log gathering. This issue resulted in depending on timing the log gathering operation being interrupted prematurely, leading to partial log collection. This is now fixed by adding the correct wait in the must-gather logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906355[*BZ#1906355*])
```

Comment:
This feels run-on-y with the "depending on timing" in the middle there. I'd add commas around that at the very least.